lib/features/power/implementations/crelay/index.ts

```diff
@@ -35,18 +44,17 @@ export class Crelay implements Power {
     // Power on the DUT
     async on(voltage?: number): Promise<void> {
         console.log(`Powering on DUT...`);
-        await execAsync(`crelay ${this.relayId} ${this.connOn}`);
+        await execAsync(`${this.crelayCmd} ${this.connOn}`);
```

Comment:
`connOn` doesn't give a lot of information what this variable would be? Suggest changing it to something more descriptive.